### PR TITLE
MF:249 Fixed error while importing config.js from frontend directory

### PR DESCRIPTION
**Pull request by donaldkibet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -449,13 +460,13 @@ Name | Type |
...
 
-*Defined in [validators/validators.ts:21](https://github.com/openmrs/openmrs-esm-module-config/blob/9766947/src/validators/validators.ts#L21)*
+*Defined in [validators/validators.ts:21](https://github.com/donaldkibet/openmrs-esm-module-config/blob/0f99e76/src/validators/validators.ts#L21)*
 
 ###  isBoolean
 
 • **isBoolean**: *function*
 
-*Defined in [validators/validators.ts:21](https://github.com/openmrs/openmrs-esm-module-config/blob/9766947/src/validators/validators.ts#L21)*
+*Defined in [validators/validators.ts:21](https://github.com/donaldkibet/openmrs-esm-module-config/blob/0f99e76/src/validators/validators.ts#L21)*
```
Comment (FlorianRappl) [inline README.md:469]: I guess all of them ran automatically, however, I think we should keep them from the `openmrs` organization (primary) and not from the fork (secondary).

Any other thoughts here? (cc @brandones )

File: README.md [hunk 2]
```diff
@@ -54,12 +54,23 @@ This is the method to use if you do not have an
...
+    "@openmrs/esm-login-app": {
+      "chooseLocation": {
+        "enabled": true
+      }
+    }
+  }
+}
 
-```json
+```import json
```
Comment (FlorianRappl) [inline README.md:70]: Why import in front? (I actually don't know what that does)

Comment (donaldkibet) [reply README.md]: specifying the file name.

File: README.md (hunk 1 above)
Comment (donaldkibet) [reply README.md]: yeap it ran automatically fixed it

File: README.md (hunk 2 above)
Comment (FlorianRappl) [reply README.md]: Ah okay - but it is not shown (rendered) by GitHub, right? So only interesting for the MD reader, right?

Comment (donaldkibet) [reply README.md]: its upto date now

**Pull request merged**